pages/account/manage/links.js

```diff
@@ -45,6 +50,12 @@ export default function ManageLinks({ BASE_URL, username, links }) {
       />
 
       <Page>
+        {success ? (
+          <Alert
+            type="success"
+            message={success ? "Link Created/Updated Successfully" : ""}
```

Comment:
I don't think the ternary here is needed because the component is not displayed when `false`